### PR TITLE
Fix QR code margin for mobile

### LINKED ISSUE #2459: No gap between QR and select tabs
### Description

Looks fine on the desktop as there's no top border on the tab list on desktop 

### Screenshots

image

### Additional information

can have a little gap at the bottom of the QR on mobile size

### PR DESCRIPTION
**Pull request by Muthukumar-Muthu**
## Fixes Issue

fixes #2459

The vertical gap between the QR code and select on the profile page
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2468"></a>

### CODE REVIEW

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍 

Just a note for next time, it would be great if you can add a "before" and "after" screenshot

**Comment by Aadarsh805** [comment]
@eddiejaoude that change won't fix the issue on screen size under 'sm'

File: components/user/UserProfile.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default function UserProfile({ BASE_URL, data }) {
         </div>
       )}
 
-      <div className="flex justify-center">
+      <div className="flex justify-center sm:my-4">
```
Comment (Aadarsh805) [inline components/user/UserProfile.js:75]: should be: <div className="flex justify-center my-4">

**Comment by eddiejaoude** [comment]
> that change won't fix the issue on screen size under 'sm'

@Aadarsh805 good point 👍 would you like to submit a new PR to improve it further?

**Comment by Aadarsh805** [comment]
@yrs147 is on it

**Pull request merged**